mac/Info-xcode.plist

```diff
@@ -27,7 +27,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleIconFile</key>
-	<string>mainicon.icns</string>
+	<string>mainicon.icns</string> <!-- Hopefully, this should not need the path to the file since we include it via qmake -->
```

Comment:
Needs to be verified.

Reply:
The autobuild mac version is OK - at least no images are missing as far as I can tell